parsec/remote_dep_mpi.c

```diff
@@ -929,13 +927,14 @@ remote_dep_get_datatypes(parsec_execution_stream_t* es,
             origin->msg.task_class_id = dtd_task->super.task_class->task_class_id;
```

Comment:
In the debug printout above we still use local_mask. I don't really understand why we now use `(1U<<k)` on line 930, but the print should be consistent with that change

Reply:
This code has been copied from the PTG, where we do have the opportunity to handle multiple deps in same time, which is exactly what the mask identifies. However, in DTD there is no such thing as being able to call iterate_successors with a mask that has more than one active bits. So, while the local_mask contains (1U<<k), I think it is more clear to not use it in iterate_successors, but instead highlight that we call it with a single enabled bit.